crates/ibc-testkit/Cargo.toml

```diff
@@ -1,19 +1,47 @@
 [package]
 name         = "ibc-testkit"
-version      = "0.47.0"
-edition      = "2021"
+version      = { workspace = true }
+authors      = { workspace = true }
+edition      = { workspace = true }
+rust-version = { workspace = true }
+license      = { workspace = true }
+repository   = { workspace = true }
 readme       = "README.md"
 keywords     = ["blockchain", "consensus", "cosmos", "ibc", "tendermint"]
-repository   = "https://github.com/cosmos/ibc-rs"
-authors      = ["Informal Systems <hello@informal.systems>"]
-rust-version = "1.60"
 description  = """
     `ibc-testkit` is a versatile library that provides essential abstractions and implementations, 
     fulfilling a dual role of enabling rigorous integration testing for the `ibc-rs` implementation 
     while also aiding host chains in addressing a broad spectrum of testing scenarios during 
     their integration with `ibc-rs`.
 """
 
+[dependencies]
+# external dependencies
+bytes = { workspace = true }
+derive_more = { workspace = true }
+displaydoc = { workspace = true }
+parking_lot = { workspace = true }
+primitive-types = { workspace = true }
+prost = { workspace = true }
+serde = { workspace = true, optional = true }
+serde_json = { workspace = true, optional = true }
+subtle-encoding = { workspace = true }
+tracing = { workspace = true }
+typed-builder = { workspace = true }
+
+# ibc dependencies
+ibc = { version = "0.47.0" , path = "../ibc" } # NOTE: since `ibc-testkit` does not well support `no_std` yet, we keep `ibc` default features enabled
```

Comment:
When you don't specify the version it can lead to potential breaks during/after our releases. Can't quite remember, what was the issue, but I feel like the issue was when someone wants to work only with e.g. `ibc-derive` crate, when not specifying the version it doesn't have any reference to pick a proper `ibc` version.